Content.Server/StationEvents/Events/SlimesSpawn.cs

```diff
@@ -0,0 +1,36 @@
+using Content.Server.StationEvents.Components;
+using Content.Shared.Actions;
+using Robust.Shared.Random;
+using System.Linq;
+
+namespace Content.Server.StationEvents.Events;
+
+public sealed class SlimesSpawn : StationEventSystem
+{
+    public static List<string> SpawnedPrototypeChoices = new List<string>()
+        {"MobAdultSlimesBlueAngry", "MobAdultSlimesGreenAngry", "MobAdultSlimesYellowAngry"};
+
+    public override string Prototype => "SlimesSpawn";
+
+    public override void Started()
+    {
+        base.Started();
+        var spawnChoice = RobustRandom.Pick(SpawnedPrototypeChoices);
+        var spawnLocations = EntityManager.EntityQuery<VentCritterSpawnLocationComponent>().ToList();
+        RobustRandom.Shuffle(spawnLocations);
+
+        var mod = Math.Sqrt(GetSeverityModifier());
+
+        var spawnAmount = (int) (RobustRandom.Next(4, 8) * mod);
+        Sawmill.Info($"Spawning {spawnAmount} of {spawnChoice}");
+        foreach (var location in spawnLocations)
+        {
+            if (spawnAmount-- == 0)
+                break;
+
+            var transform = Transform(location.Owner);
+
+            Spawn(spawnChoice, transform.Coordinates);
+        }
+    }
+}
```

Comment:
Please don't just copypaste this, we already have 3 copies of the same code as is.

Reply:
I'm not a very good coder. That's why I used existing code variants. The only thing, I can change the number of slimes spawn. But I'll see what I can do.